arch/generic/crc32_braid_c.c

```
@@ -211,6 +207,579 @@ Z_INTERNAL uint32_t PREFIX(crc32_braid)(uint32_t crc, const uint8_t *buf, size_t
+    *((uint64_t*) (final + (0*8))) ^= next1;
+    *((uint64_t*) (final + (1*8))) ^= next2;
+    *((uint64_t*) (final + (2*8))) ^= next3;
+    *((uint64_t*) (final + (3*8))) ^= next4;
+    *((uint64_t*) (final + (4*8))) ^= next5;
+
+    crc = crc32_braid_base(crc, final, len-i);
+
+    return crc;
+}
+
+Z_INTERNAL uint32_t PREFIX(crc32_braid)(uint32_t crc, const uint8_t *buf, size_t len) {
```

cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If PREFIX is a macro then please configure it.

(unknownMacro)